src/coreclr/debug/ee/arm64/walker.cpp

```diff
@@ -280,7 +280,7 @@ BOOL  NativeWalker::DecodePCRelativeBranchInst(PT_CONTEXT context, const PRD_TYP
         // Sign extension
         if ((offset & 0x4000000)) //Check for 26'st bit
         {
-            offset = offset | 0xFFFFFFFFF8000000;
+            offset = offset | 0xFFFFFFFFF0000000;
```

Comment:
It seems to me the `offset & 0x4000000` check is wrong. 26 bit constant means the last bit is `1 << 25`, which after shift would be `1 << 27 = 0x8000000`.